src/use/use.py

```diff
@@ -630,7 +632,7 @@ def _find_latest_working_version(releases: Dict[str, List[Dict[str, str]]], # {v
                                     hash_algo:str,
                                     #testing
                                     sys_version:Version=None,
-                                    platform_tags:Set[str]=set(),
+                                    platform_tags:Set[str]=frozenset(),
```

Comment:
not sure if that solves the issue. usually in that case, you'd default to None and var = {} if var is None else var